plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -298,6 +310,51 @@ export class PolicesServer {
       resp.status(200).end();
     });
 
+    const urlReader = UrlReaders.default({
+      config: this.config,
+      logger: this.logger,
+      factories: [PluginEndpointCollector.permissionFactory],
+    });
+
+    const pec = new PluginEndpointCollector(
+      this.discovery,
+      this.pluginIdProvider,
+    );
+
+    router.get('/plugins/policies', async (req, response) => {
+      const decision = await this.authorize(
+        this.identity,
+        req,
+        this.permissions,
+        {
+          permission: policyEntityReadPermission,
+        },
+      );
+
+      if (decision.result === AuthorizeResult.DENY) {
+        throw new NotAllowedError(); // 403
+      }
+      const endpoints = await pec.get();
+      let perms: Permission[] = [];
+      for (const endpoint of endpoints) {
+        const wellKnownURL = `${endpoint}/.well-known/backstage/permissions/metadata`;
+        try {
+          const permResp = await urlReader.readUrl(wellKnownURL);
+          const permMetaDataRaw = (await permResp.buffer()).toString();
+          const permMetaData = JSON.parse(permMetaDataRaw);
+          if (permMetaData) {
+            perms = [...perms, ...permMetaData.permissions];
```

Comment:
Do we want to expand this to also return the conditional rules along with the permissions?